network/benchmarks/netperf/launch.go

```diff
@@ -93,7 +94,7 @@ func setupClient() *kubernetes.Clientset {
 
 // getMinions : Only return schedulable/worker nodes
 func getMinionNodes(c *kubernetes.Clientset) *api.NodeList {
-	nodes, err := c.Core().Nodes().List(
+	nodes, err := c.CoreV1().Nodes().List(context.TODO(),
```

Comment:
network Godeps/ are pretty old now - so to make it work, you need to update godeps (or ideally upgrade to go modules and then update)